Dart/src/com/jetbrains/lang/dart/ide/runner/server/vmService/DartVmServiceListener.java

```diff
@@ -118,6 +119,12 @@ private void onIsolatePaused(@NotNull final IsolateRef isolateRef,
 
       final XLineBreakpoint<XBreakpointProperties> xBreakpoint = myBreakpointHandler.getXBreakpoint(vmBreakpoints.get(0));
 
+      if (xBreakpoint == null) {
+        myLatestSourcePosition = sourcePosition;
+        myDebugProcess.getSession().positionReached(suspendContext);
+        return;
+      }
```

Comment:
The VM implements the async step feature by setting temporary VM-only breakpoints. When we pause, we see it as stopping at a breakpoint, but the location doesn't match up to any of our known breakopints.